theme-dev/README.md

```diff
@@ -1,33 +1,295 @@
+> Some things may apply to the XPosed Module too
 
-* [Main Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/MainStrings.md)
-* [Simple Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/SimpleStrings.md)
-* [Accent Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/AccentColors.md)
-* [Primary Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/PrimaryColors.md)
-* [uikit Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/uikitStrings.md)
-* [Drawable Strings](https://github.com/GangsterFox/documentation/blob/main/theme-dev/DrawableStrings.md)
-* [Material You](/theme-dev/MaterialYou.md)
-* [Random Things which may change over time, and also some resources which might help](https://github.com/Aliucord/documentation/blob/main/theme-dev/RandomThings.md)
+1. [Small Introduction](#small-introduction)
+    1. [Creating a Simple Theme](#creating-a-simple-theme)
+2. [Main Strings](#main-strings)
+    1. [Manifest Strings](#manifest-strings)
+    2. [Background Strings](#background-strings)
+    3. [Font Strings](#font-strings)
+    4. [Allowed URLs](#urls-allowed-to-be-used-with-themer)
+3. [Simple Strings](#simple-strings)
+4. [Accent Strings](#accent-strings)
+5. [Primary Strings](#primary-strings)
+    1. [General Strings](#general-strings)
+6. [uikit Strings](#uikit-strings)
+7. [Drawable Strings](#drawable-strings)
+8. [Material You](#material-you)
+    1. [Getting Started](#getting-started)
+    2. [Adding Material You Colors](#adding-material-you-colors)
+9. [Random Things](#random-things)
+    1. [Advanced Resources](#advanced-resources)
+    2. [Light Mode Theming](#light-mode-theming)
+<br />
+<br />
 
-<h1 align="Center">Small Introduction</h1>
+# Small Introduction
 
-<p align="center">DIRECTLY MODIFY THE JSON FILE AT YOUR OWN RISK, YOU WILL LIKELY FUCK SOMETHING UP</p>
+ **DIRECTLY MODIFY THE JSON FILE AT YOUR OWN RISK, YOU WILL LIKELY FUCK SOMETHING UP**
 
-<details>
- <summary>Creating a simple theme</summary>
+
+### Creating a Simple Theme
  
-  * Start by making a new theme inside of the themer plugin settings, give it a name and it will set version and author automatically for you.
+  * Start by making a new Theme inside of the Themer Plugin settings, give it a name and it will set Version, and Author (You) automatically for you.
  * You will see multiple categories, choose the Simple Colors category and click on the + icon to add a new string. Add a `background` string and give it some color by clicking on it. 
  * Save, then select restart to see how it looks!
- * [Example Simple Colors Theme](https://cdn.discordapp.com/attachments/824357609778708580/865289689363251210/DiscordThemer_ZelkButBasic.json)
+
+[Download Example Theme](https://cdn.discordapp.com/attachments/824357609778708580/865289689363251210/DiscordThemer_ZelkButBasic.json)
 </details>
```

Comment:
what is this here for :guh:

Reply:
Show a way these strings can be used. I didn't add this specifc one though.